docs/static/monitoring/monitoring-mb.asciidoc

```diff
@@ -0,0 +1,203 @@
+
+--
+// tag::disable-ls-collection[]
+The `monitoring` setting is in the {ls} configuration file (+{ls}.yml+), but is
+commented out: 
+
+[source,yaml]
+----------------------------------
+monitoring.enabled: false
+----------------------------------
+
+Remove the `#` at the beginning of the line to enable the setting.
+// end::disable-ls-collection[]
+
+--
+
+
+[float]
+[[configure-metricbeat]]
+==== Install and configure {metricbeat}
+
+. {metricbeat-ref}/metricbeat-installation.html[Install {metricbeat}] on the
+same server as {ls}. 
+
+. Enable the `logstash-xpack` module in {metricbeat}. +
++
+--
+// tag::enable-ls-module[]
+To enable the default configuration in the {metricbeat} `modules.d` directory, 
+run: 
+
+*deb, rpm, or brew:* +
+
+["source","sh",subs="attributes"]
+----
+metricbeat modules enable logstash-xpack
+----
+
+*linux or mac:*
+
+["source","sh",subs="attributes"]
+----
+./metricbeat modules enable logstash-xpack
+----
+
+*win:*
+
+["source","sh",subs="attributes"]
+----
+PS > .{backslash}metricbeat.exe modules enable logstash-xpack
+----
+
+For more information, see 
+{metricbeat-ref}/configuration-metricbeat.html[Specify which modules to run] and 
+{metricbeat-ref}/metricbeat-module-beat.html[beat module]. 
+// end::enable-beat-module[]
+--
+
+. Configure the `logstash-xpack` module in {metricbeat}. +
++
+--
+// tag::configure-beat-module[]
+The `modules.d/logstash-xpack.yml` file contains these settings:
+
+[source,yaml]
+----------------------------------
+  - module: logstash
+    metricsets:
+      - node
+      - node_stats
+    period: 10s
+    hosts: ["localhost:9600"]
+    #username: "user"
+    #password: "secret"
+    xpack.enabled: true
+----------------------------------
+ 
+Set the `hosts`, `username`, and `password` settings as required by your
+environment. For other module settings, it's recommended that you accept the
+defaults.
+
+By default, the module collects {ls} monitoring data from
+`localhost:9600`. 
+
+To monitor multiple {ls} instances, specify a list of hosts, for example:
+[source,yaml]
+----------------------------------
+hosts: ["http://localhost:9601","http://localhost:9602","http://localhost:9603"]
+----------------------------------
+
+*Encrypted communications.* If you configured {ls} to use encrypted communications, you must access
+it using HTTPS. For example, use a `hosts` setting like `https://localhost:9600`.
+// end::configure-ls-module[]
+
+// tag::remote-monitoring-user[]
+*Elastic security.* If the Elastic {security-features} are enabled, provide a user 
+ID and password so that {metricbeat} can collect metrics successfully: 
+
+.. Create a user on the production cluster that has the 
+`remote_monitoring_collector` {stack-ov}/built-in-roles.html[built-in role]. 
+Alternatively, use the `remote_monitoring_user` 
+{stack-ov}/built-in-users.html[built-in user].
+
+.. Add the `username` and `password` settings to the beat module configuration 
```

Comment:
Here should we specify that we are still talking about the `logstash-xpack.yml` file?